feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/MetaAccountGroupingInteractorImpl.kt

```diff
@@ -67,6 +72,27 @@ class MetaAccountGroupingInteractorImpl(
             .toSortedMap(metaAccountTypeComparator())
     }
 
+    override fun updatedProxieds(): Flow<GroupedList<LightMetaAccount.Status, ProxiedAndProxyMetaAccount>> {
+        return combine(
+            metaAccountsUpdatesRegistry.observeUpdates(),
+            accountRepository.allMetaAccountsFlow(),
+            chainRegistry.chainsById
+        ) { updatedMetaIds, metaAccount, chainsById ->
+            val metaById = metaAccount.associateBy(MetaAccount::id)
+            metaAccount
+                .filter { it.type == LightMetaAccount.Type.PROXIED && updatedMetaIds.contains(it.id) }
+                .map {
+                    ProxiedAndProxyMetaAccount(
+                        it,
+                        metaById[it.proxy?.metaId] ?: error("Proxy meta account not found"),
```

Comment:
Maybe just skip such accounts via `mapNotNull` to prevent crash even though situation seems impossible?

Reply:
Imgo it is better to not use exceptions for such kind of task

Reply:
Alright, I'll fix it